nixos/modules/config/gai-conf.nix

```
@@ -0,0 +1,120 @@
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+let
+  cfg = config.networking.gaiConf;
```

```suggestion
  cfg = config.networking.getaddressinfo;
```

Only module are about config, so we should probably use the function name as context. No further input from me though.